pkg/provisioner/ironic/ironic.go

```diff
@@ -931,6 +931,26 @@ func (p *ironicProvisioner) Deprovision(deleteIt bool) (result provisioner.Resul
 			nodes.ProvisionStateOpts{Target: nodes.TargetManage},
 		)
 
+	case nodes.Inpsecting:
```

Comment:
s/Inpsecting/Inspecting

Reply:
nvm, didn't see the comment above

Reply:
btw I just checked and it seems fixed in gophercloud
https://github.com/gophercloud/gophercloud/blob/master/openstack/baremetal/v1/nodes/requests.go#L37

Reply:
OK, good. I will update that in a separate PR.

Reply:
Yes, I fixed it a couple of weeks ago.